daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -36,7 +36,7 @@ dapr run [flags] [command]
 | `--help`, `-h`                 |                      |                                                                                    | Print this help message                                                                              |
 | `--image`                      |                      |                                                                                    | The image to build the code in. Input is: `repository/image`                                         |
 | `--log-level`                  |                      | `info`                                                                             | The log verbosity. Valid values are: `debug`, `info`, `warn`, `error`, `fatal`, or `panic`           |
-| `--api-log-level`                  |                      | `debug`                                                                             | The API log verbosity. Valid values are: `debug` or `info`        |
+| `--enable-api-logging`                  |                      | `false`                                                                             | The API log verbosity.        |
```

Comment:
```suggestion
| `--enable-api-logging`                  |                      | `false`                                                                             | Enable the logging of all API calls from application to Dapr.        |
```

Reply:
Can you add an example in the examples sections in the file. Like this

dapr run --enable-api-logging node myapp.js